templates/frontend/pages/contact.tpl

```diff
@@ -0,0 +1,128 @@
+{**
+ * templates/frontend/pages/contact.tpl
```

Comment:
Good idea. I found the entire `AboutContextHandler` could be moved into `pkp-lib`. https://github.com/pkp/pkp-lib/pull/1448/commits/a22e5373ad10f7d6423ba0af74b52e54a1cdbf9a